lib/util.js

```diff
@@ -114,13 +113,18 @@ function wrapRouteHandlers(route, app) {
                     if (statusCode < 100 || statusCode > 599) {
                         statusCode = 500;
                     }
-                    const statusClass = `${Math.floor(statusCode / 100)}xx`;
-                    const stat = `${path}.${req.method}.`;
-                    app.metrics.endTiming([
-                        stat + statusCode,
-                        stat + statusClass,
-                        `${stat}ALL`
-                    ], startTime);
+                    app.metrics.makeMetric({
```

Comment:
Do we need to make a metric every time or can we create this necessary metric once and then only do the endtiming on every request? you'd create the metric somewhere on line 106?

Reply:
You are correct, we don't have to make a metric every time.